### DISCUSSION TITLE
Automation/script grouping

### DISCUSSION THREAD

**Discussion by Salamandar** [home-assistant/frontend#8106] [Configuration]
When you start having : 
* 2 automations turn_on + turn_off light on mouvement sensor for every room 
* 2 automations turn_on + turn_off light for sunrise+sunset
* Automations for the terrarium temperature
* ...

The automations list starts to be a little bloated.

Grouping them would be great. Visually, I mean, with no feature behind this. Maybe with a groupped enable-disable button.

**Comment by sim-san** [comment]
I can imagine having folders into which I can drag and drop the automations.

**Comment by KTibow** [comment]
Related WTH: https://community.home-assistant.io/t/put-automations-scripts-in-folders-groups/219510?u=ktibow
